components/prism-io.js

```diff
@@ -0,0 +1,31 @@
+Prism.languages.io = {
+    'comment': [
+        {
+            pattern: /(^|[^\\])\/\*[\s\S]*?(?:\*\/|$)/,
+            lookbehind: true
+        },
+        {
+            pattern: /(^|[^\\:])\/\/.*/,
+            lookbehind: true
+        },
+        {
+            pattern: /(^|[^\\:])#.*/,
+            lookbehind: true
+        }
+    ],
+    'triple-quoted-string': {
+        pattern: /"""(?:\\.|(?!""")[^\\])*"""/,
```

Comment:
Are you sure line terminators can't be escaped? Cause if they are you need to replace `\\.` with `\\[\s\S]`. Please double check.